test/ui-test/src/common/Common.py

```diff
@@ -1,16 +1,15 @@
 
-from drivers.SquishDriver import *
+import drivers.SquishDriver as driver
 from drivers.SquishDriverVerification import *
```

Comment:
nit: could do it here too

Reply:
What do you mean?! :)

Reply:
I mean you could also name the import:
`import drivers.SquishDriverVerification as verification`

Naming import always help on the long run :)

Reply:
Ah, yes!! sure thing!! I totally agree! :)